conda/bld.bat

```diff
@@ -7,6 +7,7 @@ move %INSTALL_PREFIX%\bin\units-shared.dll %CONDA_PREFIX%\bin\
 move %INSTALL_PREFIX%\Lib\scipp-*.lib %CONDA_PREFIX%\Lib\
 move %INSTALL_PREFIX%\Lib\units-shared.lib %CONDA_PREFIX%\Lib\
 move %INSTALL_PREFIX%\Lib\cmake\scipp %CONDA_PREFIX%\Lib\cmake\
+move %INSTALL_PREFIX%\include\scipp %CONDA_PREFIX%\include\
 move %INSTALL_PREFIX%\include\scipp* %CONDA_PREFIX%\include\
```

Comment:
To avoid the confusion, should we have here something like `scipp*.*` or `scipp-*` or something to differentiate the two?